src/test/java/edu/ksu/canvas/attendance/form/CourseConfigurationValidatorUTest.java

```diff
@@ -52,4 +53,84 @@ public void validate_rejectMinutesPerSessionGreaterThanTotalMinutes() {
+        courseConfigurationForm.setGradingOn(true);
+        courseConfigurationForm.setAssignmentPoints(100.0);
+        courseConfigurationForm.setPresentPoints(100.0);
+        courseConfigurationForm.setAbsentPoints(120.0);
+        courseConfigurationForm.setExcusedPoints(0.0);
+        courseConfigurationForm.setTardyPoints(0.0);
+
+        ArgumentCaptor<String> capturedField = ArgumentCaptor.forClass(String.class);
+        ArgumentCaptor<String> capturedErrorCode = ArgumentCaptor.forClass(String.class);
+
+        courseConfigurationValidator.validate(courseConfigurationForm,errors);
+
+        verify(errors, times(1)).rejectValue(capturedField.capture(), capturedErrorCode.capture());
+        assertEquals(1, capturedErrorCode.getAllValues().size());
+        assertEquals("Point values must be set between 0 and 100.", capturedErrorCode.getValue());
+    }
+
+    @Test
+    public void validate_rejectNullName() {
+        courseConfigurationForm.setAssignmentName(null);
+        courseConfigurationForm.setGradingOn(true);
+        courseConfigurationForm.setAssignmentPoints(100.0);
+        courseConfigurationForm.setPresentPoints(100.0);
+        courseConfigurationForm.setAbsentPoints(100.0);
+        courseConfigurationForm.setExcusedPoints(0.0);
+        courseConfigurationForm.setTardyPoints(0.0);
+
+        ArgumentCaptor<String> capturedField = ArgumentCaptor.forClass(String.class);
+        ArgumentCaptor<String> capturedErrorCode = ArgumentCaptor.forClass(String.class);
+
+        courseConfigurationValidator.validate(courseConfigurationForm,errors);
+
+        verify(errors, times(1)).rejectValue(capturedField.capture(), capturedErrorCode.capture());
+        assertEquals(1, capturedErrorCode.getAllValues().size());
+        assertEquals("Assignment Name is required.", capturedErrorCode.getValue());
+    }
+
+    @Test
+    public void validate_rejectBadAssignmentPoints() {
+        courseConfigurationForm.setAssignmentName("Attendance Assignment");
+        courseConfigurationForm.setGradingOn(true);
+        courseConfigurationForm.setAssignmentPoints(-100.0);
+        courseConfigurationForm.setPresentPoints(100.0);
+        courseConfigurationForm.setAbsentPoints(100.0);
+        courseConfigurationForm.setExcusedPoints(0.0);
+        courseConfigurationForm.setTardyPoints(0.0);
+
+        ArgumentCaptor<String> capturedField = ArgumentCaptor.forClass(String.class);
+        ArgumentCaptor<String> capturedErrorCode = ArgumentCaptor.forClass(String.class);
+
+        courseConfigurationValidator.validate(courseConfigurationForm,errors);
+
+        verify(errors, times(1)).rejectValue(capturedField.capture(), capturedErrorCode.capture());
+        assertEquals(1, capturedErrorCode.getAllValues().size());
+        assertEquals("Total Points is a required field and must be greater than 0.", capturedErrorCode.getValue());
+    }
+
+    @Test
+    public void validate_rejectNullPoints() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS00100)